src/Components/WebAssembly/testassets/Wasm.Authentication.Server/Wasm.Authentication.Server.csproj

```diff
@@ -17,11 +17,11 @@
     <Reference Include="Microsoft.AspNetCore.Components.WebAssembly.Server" />
     <Reference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" />
     <Reference Include="Microsoft.AspNetCore.Authentication.Cookies" />
-    <Reference Include="Microsoft.EntityFrameworkCore.Design" />
     <Reference Include="Microsoft.AspNetCore.Identity.UI" />
     <Reference Include="Microsoft.EntityFrameworkCore.Relational" />
     <Reference Include="Microsoft.EntityFrameworkCore.SQLite" />
     <Reference Include="Microsoft.Extensions.Hosting" />
+    <Reference Include="Microsoft.EntityFrameworkCore.Tools"  />
```

Comment:
Should we also add `microsoft.aspnetcore.components.webassembly.devserver`?

https://github.com/dotnet/runtime/issues/58667#issuecomment-912898439

Reply:
nope, this is the server of a hosted app